include/trampolines.h

```diff
@@ -77,6 +77,14 @@ dmtcp_setup_trampoline_by_addr(void *addr,
                                void *trampoline_fn,
                                trampoline_info_t *info)
 {
+  /* Trick to get "free" conversion of a long value to the
+     character-array representation of that value. Different sizes of
+     long and endian-ness are handled automatically. */
+  union u {
+    void *val;
+    char bytes[sizeof(void *)];
+  } data;
```

Comment:
@JainTwinkle,
    I know that the use of `data.bytes` in `memcpy()` existed even before your code.  But can we clean up that very old code?  It seems to me that:<br>
`memcpy(info->jump + ADDR_OFFSET, &trampoline_fn, sizeof(void *));`<br>
will do exactly the same thing as this older code:
`memcpy(info->jump + ADDR_OFFSET, data.bytes, sizeof(data.bytes));`

If you agree with this modification, then we can finally remove the above lines (`/* Trick to ... union u {...} data;`.
And if you agree with these modifications, then could you create this as a second, separate commit inside this same PR (or a separate PR, if you prefer)?  Thanks.

Reply:
Hi @JainTwinkle, Let's discuss the endianness issue face-to-face.  It will be easier for these subtle questions.  I think the revised version will produce the correct endianness, but maybe I'm wrong -- and so it's easier to talk directly.  Anyway, the current PR is good, and that discussion will be about a future PR.